README.md

```diff
@@ -106,10 +106,12 @@ helm:
       url: https://example.com/my-private-repo/
 releases:
   - name: cluster-autoscaler  # Specify the release name
+    deployment: kubectl # REQUIRED: Specify how the chart should be installed helm/kubectl
```

Comment:
Can you add some documentation about the different deployment modes elsewhere in the readme? Other users wouldn't know what this is without reading the source code.

Reply:
Fixed: 0395f1b01df2ee0ec09afe75356be7141dea7a38

Not sure if you want the documentation in a different spot. Let me know